### PR TITLE
Document the correct Spark client version (0.13.0, maybe assembled)

### LINKED ISSUE #7707: [Bug]: Docs point at old Spark metadata client version
### What happened?

What actually happened, including error codes if applies.
        
Steps to Reproduce:
1. Go to https://docs.lakefs.io/reference/spark-client.html
1. Docs say to install 0.11.0

### Expected behavior

The latest version should appear (0.13.0), and an explicit recommendation to use the assembled version on DataBricks

### PR DESCRIPTION
**Pull request by arielshaqed**
Fixes #7707.

### CODE REVIEW

File: docs/reference/spark-client.md [hunk 1]
```diff
@@ -19,17 +19,30 @@ Utilize the power of Spark to interact with the metadata on lakeFS. Possible use
...
-```
-
-Alternatively an assembled jar is available on S3, at
-`s3://treeverse-clients-us-east/lakefs-spark-client/0.11.0/lakefs-spark-client-assembly-0.11.0.jar`
+The Spark metadata client is compiled for Spark 3.1.2 with Hadoop 3.2.1, but
+can work for other Spark versions and higher Hadoop versions.
+
+<div class="tabs">
+  <ul>
+    <li><a href="#spark-shell">`pyspark`, `spark-shell`, `spark-submit`, `spark-sql`</a></li>
```
Comment (itaiad200) [inline docs/reference/spark-client.md:27]: image

Not the best looking title, maybe 
`pyspark / spark-shell / spark-submit / spark-sql`

**Review by itaiad200 — APPROVED** [review]
LGTM
Single non-blocking comment

File: docs/reference/spark-client.md (hunk 1 above)
Comment (arielshaqed) [reply docs/reference/spark-client.md]: Yikes!

Fixed, thanks!

**Comment by arielshaqed** [comment]
Thanks!

Pulling...

**Comment by arielshaqed** [comment]
(Force-pull, this is docs)

**Pull request merged**